### DISCUSSION TITLE
Any plan for release soon for mux ?

### DISCUSSION THREAD

**Discussion by aman00323** [gorilla/.github#20] [General]
- We are looking for the change from gorilla mux router to other router. But since new core team has picked up i am curious to know if there will be any release ? if yes it will be helpful if you gave some expected time frame for the same.

Also i would like to appreciate new members for taking responsibility :bow:

**Comment by coreydaley** [comment]
Thanks for the question!
We have two repositories left to transition that we hope to have completed by the end of next week. We would also like to address the majority of the open pull requests and any high priority issues that are logged for the repositories. After that we will go through and create new releases for all projects. I am hoping that we will have things ready in the next couple of months.

We will also start creating some milestones soon as we triage all of the existing issues and pull requests.

**Comment by coreydaley** [comment]
We have released v2.0 for all of the repositories, here is a link to the mux release: https://github.com/gorilla/mux/releases/tag/v2.0
